plugin/trino-hive/src/main/java/io/trino/plugin/hive/procedure/RegisterPartitionProcedure.java

```diff
@@ -103,6 +104,11 @@ public void registerPartition(ConnectorSession session, ConnectorAccessControl a
 
     private void doRegisterPartition(ConnectorSession session, ConnectorAccessControl accessControl, String schemaName, String tableName, List<String> partitionColumn, List<String> partitionValues, String location)
     {
+        checkProcedureArgument(schemaName != null, "schema_name cannot be null");
+        checkProcedureArgument(tableName != null, "table_name cannot be null");
+        checkProcedureArgument(partitionColumn != null, "partition_columns cannot be null");
```

Comment:
Could you rename `partitionColumn` to `partitionColumns` in a separated commit or PR?